src/main/java/zeenea/connector/SourceItem.java

```diff
@@ -0,0 +1,403 @@
+package zeenea.connector;
+
+import java.math.BigDecimal;
+import java.net.URI;
+import java.time.Instant;
+import java.util.*;
+import zeenea.connector.common.ItemIdentifier;
+import zeenea.connector.contact.SourceContactRelation;
+import zeenea.connector.property.*;
+
+/**
+ * Base class for all kinds of source items. An item is something that is managed and searchable in
+ * Zeenea Data Catalog.
+ *
+ * @since 1.0.0
+ */
+public abstract class SourceItem {
```

Comment:
Proposition : retirer le préfixe `Source` partout.
```suggestion
public abstract class Item {
```

Reply:
Finalement j'ai supprimé le préfixe, c'est plus lisible :)